appveyor.yml

```diff
@@ -1,3 +1,19 @@
+#
+# Copyright (C) 2016 Red Hat, Inc.
```

Comment:
Agree with you @KurtStam, no reason to update copyright I don't think. 25 years or whatever it is should be enough.